flink-runtime-web/web-dashboard/.eslintrc.js

```diff
@@ -126,7 +126,9 @@ module.exports = {
     {
       files: ['*.html'],
       extends: ['plugin:@angular-eslint/template/recommended'],
-      rules: {}
+      rules: {
+        '@angular-eslint/template/eqeqeq': 'off'
```

Comment:
I see there existing other places to use `===` in the web code, can we make the rule as `warn` instead of `off` as [doc](https://cn.eslint.org/docs/user-guide/configuring#configuring-rules) said.

Reply:
Thanks for your suggestion @Myasuka !